.github/workflows/optimize-ci.yml

```diff
@@ -297,17 +296,79 @@ jobs:
         with:
           archive_name: integration-tests-opensearch-dev-${{ matrix.includedGroups }}-docker
 
+  migration:
+    name: Migration
```

Comment:
I think we should call this `Upgrade`, especially now the other is called schema migration